README.md

```diff
@@ -31,13 +37,35 @@ This includes the following packages:
 * [ ] Create convinient layer wrapping gRPC and HTTP clients for Dapr
 * [ ] Flask extensions for Dapr State/Pubsub/Bindings
 
-## Developing
+## Getting started
 
 ### Prerequisites
 
 * [Install Dapr standalone mode](https://github.com/dapr/cli#install-dapr-on-your-local-machine-standalone)
 * [Install Python 3.8+](https://www.python.org/downloads/)
 
+### Install Dapr python sdk
+
+* Official package - WIP
+
+```sh
+pip install dapr
+```
+
+* Development package
+
+```sh
+pip install dapr-dev
```

Comment:
will dapr-dev and dapr merge in future? Can we have a more layered approach: a dapr package which contains auto generated client apis, a dapr-actor package which contain actor apis?

Reply:
most of python pkg splits two repository for daily build and official build so we will not. 

layered approach is quite complicated. unlike java and dotnet, python package zip python codes. so I am not sure that how much benefit we can get by splitting sdk to multiple ones.

Reply:
Tensorflow example - 
1. nightly build - https://pypi.org/project/tf-nightly/#history
2. official build - https://pypi.org/project/tensorflow/#history

Reply:
makes sense, when I searach on pypi: https://pypi.org/search/?q=dapr
it seemed confusing, can the name be changed or more text can be added to description of package?

Reply:
I changed the title of the packages.